dashboard/app/views/levels/new.html.haml

```diff
@@ -13,6 +13,7 @@
   %li= link_to 'Build a Playlab Droplet Level', new_level_path(type: 'StudioEC')
   %li= link_to 'Build a Minecraft Level', new_level_path(type: 'Craft')
   %li= link_to 'Build a Star Wars Grid Level', new_level_path(type: 'StarWarsGrid')
+  %li= link_to 'Build a Gamelab Jr Level', new_level_path(type: 'GamelabJr')
```

Comment:
Probably want to include a note that this is pre-alpha / subject to change.  Or omit the link from LB entirely for now.